examples/jaeger-tracing/front-envoy-jaeger.json

```diff
@@ -8,7 +8,7 @@
           "name": "http_connection_manager",
           "config": {
             "tracing": {
-              "operation_name": "ingress"
+              "operation_name": "egress"
```

Comment:
can you also check Zipkin sample configuration

Reply:
I've ran the zipkin example - but would be good if someone else could run both examples to confirm.

Reply:
same as the zipkin q.. should this be ingress? because front envoy is the server. what am I missing?